.travis.yml

```diff
@@ -81,8 +81,8 @@ matrix:
           env: PYTHON_VERSION=2.7 SETUP_CMD='test -a "--mpl"'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
                if [[ $PYTHON_VERSION == 2.7 ]];
-               then PIP_DEPENDENCIES='jplephem mock';
-               else PIP_DEPENDENCIES='jplephem';
+               then PIP_DEPENDENCIES='jplephem mock pytest-mpl';
+               else PIP_DEPENDENCIES='jplephem pytest-mpl';
```

Comment:
I don't think we actually use the channels in the main astropy testing at the moment, and i think I'd rather avoid adding that here, since pip is just as easy.